src/components/Input/NumericInput/NumericSlider/util.ts

```diff
@@ -0,0 +1,28 @@
+// @TODO (2025-03-03, emmbr) This should be handled a better way... This is a bit of a
+// hack and the max value is just arbitrarily chosen
+const maxAllowedExtentForSlider = 1e12;
+
+/**
+ * @function validSliderExtent
+ * @description Determines whether the given slider extent is valid based on the provided minimum and maximum values.
+ *
+ * @param {number} min - The minimum value of the slider.
+ * @param {number} max - The maximum value of the slider.
+ *
+ * @returns {boolean} Returns `true` if the extent (difference between max and min) is finite and less than the allowed maximum extent for the slider; otherwise, returns `false`.
+ *
+ * @remarks
+ * - The function ensures that the slider's range is within a valid and computable extent.
+ * - If no `min` or `max` values are set, the slider marks cannot be computed properly.
+ *
+ * @example
+ * ```typescript
+ * const isValid = validSliderExtent(0, 100); // true
+ * const isInvalid = validSliderExtent(0, Infinity); // false
+ * ```
+ */
+export function validSliderExtent(min: number, max: number) {
+  const extent = max - min;
+  // When no min/max is set, the marks for the slider cannot be nicely computed
+  return isFinite(extent) && extent < maxAllowedExtentForSlider;
+}
```

Comment:
This comment seems too elaborate compared to the function it documents. Not sure we want to maintain oxygen comments written to this extent? Especially since we don't have them this elaborated even in the main code base. 

How about simplifying it a bit to match what we do in OpenSpace: 

```suggestion
/**
 * Determines whether the given slider extent is valid based on the provided minimum and maximum values. 
 * If no `min` or `max` values are set, the slider marks cannot be computed properly.
 *
 * @param min - The minimum value of the slider.
 * @param max - The maximum value of the slider.
 *
 * @returns `true` if the extent (difference between max and min) is finite and less than the allowed maximum extent for the slider; otherwise, returns `false`.
 */
export function validSliderExtent(min: number, max: number) {
  const extent = max - min;
  // When no min/max is set, the marks for the slider cannot be nicely computed
  return isFinite(extent) && extent < maxAllowedExtentForSlider;
}
```

Reply:
Please also make sur ethat the comment adheres to outer max line width limitation.